spec/requests/providers/means/cash_incomes_controller_spec.rb

```diff
@@ -53,7 +53,7 @@
 
     it "shows the page" do
       request
-      expect(response.body).to include(I18n.t("providers.means.cash_incomes.show.page_heading"))
+      expect(response.body).to include(I18n.t("providers.means.cash_incomes.show.page_title"))
```

Comment:
**nit** feel free to ignore, but i generally try to remove these references to a i18n in favour of string literals. This is to avoid false positives and makes the expected content clearer

Reply:
Agreed, this slipped by me in duplication and complicated code changes... will address in future PR